### PR TITLE
Costumes for nuclear operatives

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: Nuclear operatives that wish to fail in style can purchase Centcom
Official and Clown costumes. They can also buy a defective chameleon
kit.
/:cl:

It's funny. Other costume ideas welcome.

- Also merges two Initializes in camo guns
- Centcom Inspector outfit does have a Centcom headset, but it has no encryption chips. The outfit also is missing an egun.
- Removes the "Stealthy" description from camo shoes in the nuke op uplink they buy the same damn shoes, they're just more expensive for immersive and plausible reasons.

### CODE REVIEW

File: code/game/objects/items/storage/uplink_kits.dm [hunk 1]
```diff
@@ -308,3 +308,18 @@
...
+	new /obj/item/device/radio/headset/headset_cent/empty(src)
+	new /obj/item/clothing/glasses/sunglasses(src)
+	new /obj/item/storage/backpack/satchel(src)
+	new /obj/item/device/pda/heads(src)
+	new /obj/item/clipboard(src)
+
+/obj/item/storage/box/syndie_kit/chameleon/broken/PopulateContents()
+	. = ..()
+	for(var/obj/item/I in contents)
+		I.AddComponent(/datum/component/constant_emp)
```
Comment (XDTM) [inline code/game/objects/items/storage/uplink_kits.dm]: The headset is gonna be permanently off

Comment (coiax) [reply code/game/objects/items/storage/uplink_kits.dm]: Well, it's defective equipment, what can I say. :D

**Comment by BeeSting12** [comment]
Leave the no slips in the uplink.

**Comment by coiax** [comment]
They still are in the uplink, I just phrased my changes poorly.

I just changed the name to be the same as the Traitor uplink one.

**Comment by awesine** [comment]
make it send the centcom ferry too

**Comment by kevinz000** [comment]
@coiax it would probably be more performant to just make the broken items constantly get scrambled, rather than having them get emped every 2 seconds.

**Comment by StreakyHaddock** [comment]
sprites?

**Comment by InsaneHyena** [comment]
Just give them their chameleon suits back.

**Comment by coiax** [comment]
I will change how the broken chameleon kit breaks. Constant EMP was a lazy choice.

File: code/modules/uplink/uplink_items.dm [hunk 2]
```diff
@@ -1328,6 +1321,27 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	cost = 20
 	cant_discount = TRUE
 
+/datum/uplink_item/badass/costumes
+	surplus = 0
+	include_modes = list(/datum/game_mode/nuclear)
+	cost = 4
+	cant_discount = TRUE
```
Comment (Cyberboss) [inline code/modules/uplink/uplink_items.dm]: I don't think there's anything preventing this particular type from showing up in the purchase list?

Comment (coiax) [reply code/modules/uplink/uplink_items.dm]: Entries with no item var won't show up.

File: code/modules/paperwork/stamps.dm [hunk 3]
```diff
@@ -109,3 +109,14 @@
...
 				break
 		change_to(stamp_type)
+
+/obj/item/stamp/chameleon/broken/Initialize()
+	. = ..()
+	START_PROCESSING(SSobj, src)
+
+/obj/item/stamp/chameleon/broken/Destroy()
+	STOP_PROCESSING(SSobj, src)
+	. = ..()
```
Comment (Cyberboss) [inline code/modules/paperwork/stamps.dm]: return ..()

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Pull request merged**